src/cmd-koji-upload

```diff
@@ -95,6 +96,10 @@ KOJI_CG_TYPES = {
 # against content types.
 RENAME_RAW = ['initramfs.img', '-kernel', "ostree-commit"]
 
+# These are compressed extensions that are used to determine if name managling
```

Comment:
Super nit: `#:` for documenting module level variables/constants. Not worth changing IMHO right now.